### PR TITLE
Updated Scenario for calendar

### PR DESCRIPTION
**Pull request by gipartha**
Updated scenario for calendar

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/policy-control-diagnostic-data-mobile.md [hunk 1]
```diff
@@ -1273,6 +1273,18 @@ action.
 - **upload_images** Confirms that the image upload action succeeded or failed.
 - **upload_voice_messages** Confirms that the voice message upload action succeeded or failed.
 - **voiceMessageUpload** Confirms that the voice message upload action succeeded or failed.
+- **cancel_channel_meeting** Confirms that the cancellation of channel meeting has succeeded or failed.
```
Comment (JohanFreelancer9) [inline Teams/policy-control-diagnostic-data-mobile.md:1276]:
```suggestion
- **cancel_channel_meeting** Confirms that the cancellation of a channel meeting has succeeded or failed.
```

File: Teams/policy-control-diagnostic-data-mobile.md [hunk 2]
```diff
@@ -1273,6 +1273,18 @@ action.
...
+- **cancel_meeting** Confirms that the cancellation of a meeting has succeeded or failed.
+- **cancel_private_meeting** Confirms that the cancellation of private meeting has succeeded or failed.
```
Comment (JohanFreelancer9) [inline Teams/policy-control-diagnostic-data-mobile.md:1278]:
```suggestion
- **cancel_private_meeting** Confirms that the cancellation of a private meeting has succeeded or failed.
```

File: Teams/policy-control-diagnostic-data-mobile.md [hunk 3]
```diff
@@ -1273,6 +1273,18 @@ action.
...
+- **edit_channel_meeting** Confirms that the edit operation of channel meeting has succeeded or failed.
```
Comment (JohanFreelancer9) [inline Teams/policy-control-diagnostic-data-mobile.md:1279]:
```suggestion
- **edit_channel_meeting** Confirms that the edit operation of a channel meeting has succeeded or failed.
```

File: Teams/policy-control-diagnostic-data-mobile.md [hunk 4]
```diff
@@ -1273,6 +1273,18 @@ action.
...
+- **edit_meeting** Confirms that the edit operation of meeting has succeeded or failed.
```
Comment (JohanFreelancer9) [inline Teams/policy-control-diagnostic-data-mobile.md:1280]:
```suggestion
- **edit_meeting** Confirms that the edit operation of a meeting has succeeded or failed.
```

File: Teams/policy-control-diagnostic-data-mobile.md [hunk 5]
```diff
@@ -1273,6 +1273,18 @@ action.
...
+- **server_fetch_agenda_view** Confirms that the calendar event sync using the Middle Tier API has succeeded or failed.
+- **server_fetch_date_picker_view** Confirms that the calendar event sync using the Outlook REST API has succeeded or failed.
+- **server_fetch_agenda_view_group** Confirms that the calendar event sync using the Middle Tier API for TFL group has succeeded or failed.
```
Comment (JohanFreelancer9) [inline Teams/policy-control-diagnostic-data-mobile.md:1283]:
```suggestion
- **server_fetch_agenda_view_group** Confirms that the calendar event sync using the Middle Tier API for the TFL group has succeeded or failed.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review. @gipartha, please commit my suggested changes.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@MicrosoftHeidi PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**